execution/execution.go

```diff
@@ -18,8 +18,23 @@ const (
 	Created Status = iota
 	InProgress
 	Completed
+	Failed
 )
 
+func (s Status) String() (r string) {
```

Comment:
We are using the status in some errors:
https://github.com/mesg-foundation/core/blob/925cd85930975f2171aa0786cb1fb4bfab643173/execution/error.go#L6-L14

Before, the error was like: `Execution status error: \x0 instead of \x1`.
Now it looks like: `Execution status error: completed instead of in progress`.

Reply:
I missed that part 👍